contracts/src/v0.8/ccip/libraries/Internal.sol

```diff
@@ -81,34 +75,6 @@ library Internal {
     uint256 proofFlagBits;
   }
 
-  /// @notice Report that is submitted by the execution DON at the execution phase.
-  /// @dev RMN depends on this struct, if changing, please notify the RMN maintainers.
-  struct ExecutionReport {
-    EVM2EVMMessage[] messages;
-    // Contains a bytes array for each message, each inner bytes array contains bytes per transferred token
-    bytes[][] offchainTokenData;
-    bytes32[] proofs;
-    uint256 proofFlagBits;
-  }
-
-  /// @notice The cross chain message that gets committed to EVM chains.
-  /// @dev RMN depends on this struct, if changing, please notify the RMN maintainers.
-  struct EVM2EVMMessage {
-    uint64 sourceChainSelector; // ────────╮ the chain selector of the source chain, note: not chainId
-    address sender; // ────────────────────╯ sender address on the source chain
-    address receiver; // ──────────────────╮ receiver address on the destination chain
-    uint64 sequenceNumber; // ─────────────╯ sequence number, not unique across lanes
-    uint256 gasLimit; //                     user supplied maximum gas amount available for dest chain execution
-    bool strict; // ───────────────────────╮ DEPRECATED
-    uint64 nonce; //                       │ nonce for this lane for this sender, not unique across senders/lanes
-    address feeToken; // ──────────────────╯ fee token
-    uint256 feeTokenAmount; //               fee token amount
-    bytes data; //                           arbitrary data payload supplied by the message sender
-    Client.EVMTokenAmount[] tokenAmounts; // array of tokens and amounts to transfer
-    bytes[] sourceTokenData; //              array of token data, one per token
-    bytes32 messageId; //                    a hash of the message data
-  }
-
```

Comment:
https://github.com/smartcontractkit/chainlink/pull/14739/files#diff-8deef24f4fc1cc3e29f8e8c9eff38771d3b66607e4f182e6c8d79feae4888935L112-L132

This part should be removed as well ?